mllib/src/main/scala/org/apache/spark/mllib/fpm/FPGrowth.scala

```diff
@@ -42,8 +53,9 @@ import org.apache.spark.storage.StorageLevel
  */
 @Since("1.3.0")
 @Experimental
-class FPGrowthModel[Item: ClassTag] @Since("1.3.0") (
-    @Since("1.3.0") val freqItemsets: RDD[FreqItemset[Item]]) extends Serializable {
+class FPGrowthModel[Item: ClassTag: TypeTag] @Since("1.3.0") (
```

Comment:
This will break API compatibility since old user code might provide a class for which a TypeTag is not available.